controllers/auth.js

```diff
@@ -0,0 +1,58 @@
+const mysql = require('mysql2');
+const { check, validationResult } = require('express-validator');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'sathish',
+  password: 'Sathish@19',
+  database: 'ecommerce',
+});
+
+exports.login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    if (!email || !password) {
+      return res.status(400).render('login', { count: 1, message: 'please provide email and password', title: 'Login' });
```

Comment:
Why 400 as response code ? Why not 403 ?

Reply:
yeah. I'll change that to 403. The 403 will say authentication credentials provided by the client is not correct. While 400 will say only bad request by the client.

Reply:
Manoj is that wrong if i use 400. Because i am using 400 when the user tries to enter without email or password. So its 
 like a client error.

Reply:
Why is it a client error ?  https://httpstatuses.com/400

Reply:
Because client is the one who haven't entered the email or password

Reply:
Client has not entered it.. its the user..  Client refers to the Browser here.

Reply:
Oh okay manoj. Sorry for misunderstood